sdk/storage/azure-storage-blob/src/samples/java/com/azure/storage/blob/specialized/PageBlobClientJavaDocCodeSnippets.java

```diff
@@ -424,4 +426,31 @@ public void copyIncrementalWithResponseCodeSnippet() {
         // END: com.azure.storage.blob.specialized.PageBlobClient.copyIncrementalWithResponse#String-String-RequestConditions-Duration-Context
     }
 
+    /**
+     * Code snippets for {@link PageBlobClient#copyIncrementalWithResponse(PageBlobCopyIncrementalOptions,
+     * Duration, Context)}
+     */
+    public void copyIncrementalWithResponseCodeSnippet2() {
+        // BEGIN: com.azure.storage.blob.specialized.PageBlobClient.copyIncrementalWithResponse#PageBlobCopyIncrementalOptions-Duration-Context
+        final String snapshot = "copy snapshot";
+        PageBlobCopyIncrementalRequestConditions destinationRequestConditions = new PageBlobCopyIncrementalRequestConditions()
+            .setIfNoneMatch("snapshotMatch");
+        Context context = new Context(key, value);
+
+        CopyStatusType statusType = client
+            .copyIncrementalWithResponse(new PageBlobCopyIncrementalOptions(url, snapshot)
+                .setRequestConditions(destinationRequestConditions), timeout, context).getValue();
+
+        if (CopyStatusType.SUCCESS == statusType) {
```

Comment:
nit: `switch` would be more natural choice here imo.

Reply:
yeah I can change that. This was just copy pasted from the original snippet